public/app/plugins/panel/heatmap-new/HeatmapPanel.tsx

```diff
@@ -135,12 +228,34 @@ export const HeatmapPanel: React.FC<HeatmapPanelProps> = ({
       <VizLayout width={width} height={height} legend={renderLegend()}>
         {(vizWidth: number, vizHeight: number) => (
           <UPlotChart config={builder} data={facets as any} width={vizWidth} height={vizHeight} timeRange={timeRange}>
-            {/*children ? children(config, alignedFrame) : null*/}
+            <ExemplarsPlugin exemplars={exemplars!} config={builder} colorPalette={palette} />
           </UPlotChart>
         )}
       </VizLayout>
       <Portal>
-        {hover && options.tooltip.show && (
+        {hover && (
```

Comment:
🤔 with exemplars, having no-tooltip does not make sense

I wonder if we should just remove that option entirely